src/Commands/DrupalCommands.php

```diff
@@ -282,8 +282,20 @@ public function settingsSetup(array $options = [
       'root' => InputOption::VALUE_REQUIRED,
     ])
     {
+        $settings_file = $options['root'].'/sites/default/settings.php';
+        $settings_local_file = $options['root'] . '/sites/default/' . $this->getConfig()->get('drupal.settings_local_file');
```

Comment:
The site path `/sites/default/` is actually configurable, we should get it from the runner configuration.